class-wposa.php

```diff
@@ -200,7 +200,9 @@ function admin_init() {
 	            $section['desc'] = '<div class="inside">' . $section['desc'] . '</div>';
 
 	            // Create the callback for description.
-	            $callback = create_function( '', 'echo "' . str_replace( '"', '\"', $section['desc'] ) . '";' );
+	            $callback = function() use ($section) {
```

Comment:
`function() use` what kind of syntax is that? Is it PHP 5.4 compatible?

Reply:
Also let me know how did you test this PR?

Reply:
I do not have php 5.4 anymore, tested on 5.6, 7.0, 7.2
I use it in 

[woocommerce-notification](https://github.com/wonder32/woocommerce-notification) | [file](https://github.com/wonder32/woocommerce-notification/blob/master/Includes/Settings.php)

syntax example [link](http://php.net/manual/en/functions.anonymous.php#example-165)